lib/Runtime/Language/SourceTextModuleRecord.h

```diff
@@ -61,6 +62,8 @@ namespace Js
 
         bool WasParsed() const { return wasParsed; }
         void SetWasParsed() { wasParsed = true; }
+        bool WasStageOne() const { return stageOne; }
```

Comment:
As above, can we name these a little more directly? `IsEvaluationPrepassComplete`, `SetEvaluationPrepassComplete`, etc. Something along those lines? Doesn't have to be prepass, just indicate the initial evaluation step was completed.